runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/HistoryTable/HistoryTable.tsx

```diff
@@ -102,6 +102,19 @@ const HistoryTable = forwardRef<any, HistoryTableProps>(({ botId, processId, sx,
         },
     }));
 
+    const handleRowExpand = (currRow) => {
```

Comment:
is this file named `*.jsx`? xd

Reply:
rhetorical question? xD
added types - done